packages/core/src/types.ts

```diff
@@ -169,7 +170,7 @@ export type OutputFrom<T> = T extends ActorLogic<
   infer _TEmitted
 >
   ? (TSnapshot & { status: 'done' })['output']
-  : T extends ActorRef<infer TSnapshot, infer _TEvent>
+  : T extends ActorRef<infer TSnapshot, infer _TEvent, infer _TEmitted>
```

Comment:
it looks like these unused type args are just a stylistic convention?

Reply:
Not convention; without this, the alternative is `<infer Something, any, any>` and those `any`s can actually break inference because they may not match with whatever you're checking with `T extends ...`. So by using `<infer Something, infer _Unused, ...>`, you're saying "I don't care what that type is (more broad than `any`); I'm not going to use it anyway"